litmus-portal/frontend/src/views/Community/GeoMap/CountryMap.tsx

```diff
@@ -8,11 +8,14 @@ import useStyles from './styles';
 respective count of users wise to present it on map */
 
 const CountryMap = () => {
-  const data: string[][] = useSelector(
-    (state: RootState) => state.communityData.google.geoCountry
+  const { communityData } = useSelector(
+    (state: RootState) => state.communityData
   );
   const classes = useStyles();
   const theme = useTheme();
+
+  const data: string[][] = communityData.google.geoCountry;
```

Comment:
the type should be automatically inferred...
`string[][]` is not required

Reply:
done